guides/README.md

```diff
@@ -0,0 +1,16 @@
+# Guides for leto-modelizer-plugin-core
+
+## Docs
+
+In this [docs section](https://github.com/ditrit/leto-modelizer-plugin-core/blob/main/guides/docs/guide.md), you will learn how plugin-core works overall and a detailed view of all the classes and methods you will find in this project.
+
+## Migrations
+
+For each new version of plugin-core, you will have access to the [migrations guide](https://github.com/ditrit/leto-modelizer-plugin-core/blob/main/guides/migrations). So that you know what has changed, added or removed from one version to another with example code.
```

Comment:
(https://github.com/ditrit/leto-modelizer-plugin-core/blob/main/guides/migrations), to know what has been changed, added or removed from one version to another, with examples.

Reply:
done